### DISCUSSION TITLE
[Question] will AR1808FCHNENCY work with this integration?

### DISCUSSION THREAD

**Discussion by wez** [omerfaruk-aran/esphome_samsung_hvac_bus#240] [Questions]
### Your Question

I'm unsure whether this model speaks the right protocol

### Additional Context

The compatibility tool reports:

```
Model: AR1808FCHNENCY
Classification: AR
Capacity: 180 kW
Product Type: F (Other)
Product Notation: C
Feature: H
Rating Voltage: N
Mode: E
```

Is Other `Non-NASA`, or something unknown? The docs are not explicit about how to interpret or how to proceed based on the output from the tool.

This particular model has integrated smart things support, which is what I'd like to eliminate from my local control situation.  Will it coexist peacefully with the esphome controller?

Thanks!

### Pre-Submission Checklist

- [X] I have searched for a similar question in open/closed issues.
- [X] I have checked the project’s documentation for relevant information.
- [X] I have provided as much detail as possible to help answer my question.

### HVAC System Type

Other

### Samsung HVAC Device Model

AR1808FCHNENCY

### ESP Device Model

None yet

**Comment by omerfaruk-aran** [comment]
Hi,

Thank you for your question! I couldn't find any information about the model AR1808FCHNENCY in the known Samsung device database. If you have the user manual or any technical documentation for this device, I recommend checking whether it supports either the NASA or NON-NASA protocol.

If your device falls under the NASA or NON-NASA categories, it should be compatible with this integration. However, if it's classified as Other, there's a 99% chance it won't work as this category generally indicates unsupported protocols.

Let us know if you find more details about the device, and we'll be happy to assist further!

Best regards.
